code/api/views.py

```diff
@@ -186,23 +186,41 @@ class LikedView(View):
     def get(self, request, author_id):
         """ GET - Get a list of like objects from {author_id} """
         try:
+            page = request.GET.get("page")
+            size = request.GET.get("size")
+
             author = LocalAuthor.objects.get(id=author_id)
-            authorLikedPosts = LocalPost.objects.filter(likes__exact=author)
-            host = request.get_host()
+            author_liked_posts = LocalPost.objects.filter(
+                likes__author=author,
+                visibility=LocalPost.Visibility.PUBLIC
+            )
+
+            author_liked_comments = Comment.objects.filter(likes__author=author)
             likes = []
-            for post in authorLikedPosts:
+            for post in author_liked_posts:
                 like = {
                     "@context": "https://www.w3.org/ns/activitystreams",
                     "summary": f"{author.displayName} Likes your post",
                     "type": "like",
                     "author": author.as_json(),
-                    "object": f"{SCHEME}://{host}/author/{post.author.id}/posts/{post.id}"
+                    "object": f"{SCHEME}://{HOST}/author/{post.author.id}/posts/{post.id}"
+                }
+                likes.append(like)
+
+            for comment in author_liked_comments:
+                like = {
+                    "@context": "https://www.w3.org/ns/activitystreams",
+                    "summary": f"{author.displayName} Likes your comment",
+                    "type": "like",
+                    "author": author.as_json(),
+                    "object": f"{SCHEME}://{HOST}/author/{comment.post.author.id}/posts/{comment.post.id}/comments/{comment.id}"
```

Comment:
The URLs are missing the app prefix, but I think it was like that before too. This occurs in line 477 as well. Could you fix it?

In retrospect, I think we should define API_BASE = f"{SCHEME}://{HOST}/{API_PREFIX}" in constants and have every URL use that, so that we don't make mistakes writing out the URL base. Thoughts?

Reply:
That's a good catch! I like the idea of defining an API_BASE constant and have every URL use that. Thank you 🙏

Reply:
Added API_BASE